src/s3b_the_roach_market.tw2

```diff
@@ -0,0 +1,280 @@
+
+> [[Head to Auntie’s gift shop|RM-giftshop-entrance]]
+> [[Head to the junk heap|RM-junkheap-entrance]]
+
+::RM-DopemanDan-free-drugs
+Dan cackled. “Hell yeah! Now you’re talking words Dopeman understands.” He thrust the bag forward, and I took it.
+
+I unrolled the top and peeked inside. “Wow. This is, uh, a pretty full bag.”
+
+“Yeah, dawg. I figure she was either getting herself set up for a sweet-ass month or one fuckin’ buckwild night.”
+
+I was about to ask Dan if this was a regular thing for Anna, but then I thought better of it. I didn’t want him to think he had anything to say about her that I didn’t already know.
+
+“Well, cool, man. With a haul like this, I bet she’d never notice if a little went missing.” I winked at him.
+
+Dan tapped the side of his nose with his finger. “Trip in peace, my ‘lette. Dopeman ain’t never snitched on nobody.”
+
+I nodded, and made my way deeper into the Roach Market.
+
+> [[Head to Auntie’s gift shop|RM-giftshop-entrance]]
+> [[Head to the junk heap|RM-junkheap-entrance]]
+
+<!-- If using debug menu shortcuts, prompts player to enter their name before beginning Auntie's gift shop sequence -->
+::RM-giftshop-entrance-nameprompt
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 1)[Right, my name is $playerName. [[Moving on|RM-giftshop-entrance]].<!-- what this code does is check the ending of the custom $playerName string to hopefully pluralize it properly -->{
+  (if: $playerName's last is "s")[
+    (set: $playerNames to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "s"))[
+    (set: $playerNames to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "c"))[
+    (set: $playerNames to ($playerName + "es"))
+  ]
+  (else:)[
+    (set: $playerNames to ($playerName + "s"))
+  ]
+}]
+}(else:)[No, [[that's not right|RM-giftshop-entrance-nameprompt]].]
+
+::RM-giftshop-entrance
+{(set: $spokeToToe to true) <!-- forcing this to true for initial testing -->
+(if: $giftshopVisited is true)[
+<!-- alternate intro if revisiting the gift shop scene -->I was back in Auntie’s: her combination gift shop, living room, and bedroom, with cheap wire racks all around. Auntie was plopped in the middle of it all just like before, watching TV from her recliner. I think <i>Big Money Rustlas</i> was on. She loved the oldies.
+]
+(else:)[
+<!-- gift shop intro on first visit -->
+  (set: $giftshopVisited to true)
+Auntie’s “gift shop” was also her living room. And bedroom, I guess, since she usually fell asleep in the threadbare recliner she had set up in a clearing in the middle of everything. She kept the merchandise in cheap wire shelves standing in neat rows all around her, and she kept herself plopped in a chair watching a flickery TV. She had an end table on her right side, where a lamp with a yellowing shade sat on an honest-to-god white doily.
+<p>
```

Comment:
I'm not 100% on this, but my instinct is that `<br /><br />` is more in keeping with HTML standards than a `<p>` tag by itself. 

If we decide to stick with `<p>` tags, definitely include the closing `</p>` immediately afterward — some browsers may not do so automatically.

Reply:
Oops, make that `<br>`. They went and [changed the gosh-dang rules of the internet](https://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br) on me.

Reply:
👍 ya, the messy `<p>` tags I jammed in to break the compressed lines back up def don't need to stay